spec/helpers/parameters_helper_spec.rb

```diff
@@ -5,4 +5,6 @@
 RSpec.describe ParametersHelper do
   it { expect(filename_to_title('my_name_is_bob')).to eq 'My Name Is Bob' }
   it { expect(name_of_act('my/name/is/bob')).to eq 'Is' }
+  it { expect(github_file_url('./tmp/development-openfisca-aotearoa/openfisca_aotearoa/parameters/acc/weekly_compensation_abatement.yaml')).to eq 'https://github.com/ServiceInnovationLab/openfisca-aotearoa/blob/master/openfisca_aotearoa/parameters/acc/weekly_compensation_abatement.yaml' }
```

Comment:
will these tests start failing if we rename this file?

Reply:
We could instead have a single yaml file that we keep in raputure, under `spec/data` and we check we can parse that file.